#### Manuscript #2: [*GigaScience* Data Note](http://www.gigasciencejournal.com/authors/instructions/datanote)

- Detailed specification of the first CAMI challenge datasets
  - Toy: Low (single), Medium (differential-abundance), High (time-series)
    - Simulated using publicly available genomes, for method developers to prepare for the challenge
  - CAMI #1: Low (single), Medium (differential-abundance), High (time-series)
    - Unpublished genomes, specifically sequenced for the CAMI challenge
- Permanent archive of simulated reads in [GigaDB](http://gigadb.org/)
  - Scott Edmunds: "[...] this is very much of the scope, scale and utility (hosting data and pipelines that may not have a practical home) that we are aiming for with *GigaScience*. They should be very useful benchmarking datasets in the future, so its crazy you can't deposit them anywhere else."

---

# Title

Task force (alphabetically):
- [Andreas Bremges](https://github.com/abremges)
- [Peter Hofmann](https://github.com/p-hofmann)
- [Alice McHardy](https://github.com/alicemchardy)
- [Alex Sczyrba](https://github.com/asczyrba)

Co-authors (alphabetically):
- **All data donors**
- Thomas Rattei
- ...

### Abstract

**Background**
a presentation of the interest or relevance of these data for the broader community

**Findings**
a very brief preview of the data type(s) produced, the methods used, and information relevant to data validation

**Conclusions**
a short summary of the potential uses of these data and implications for the field. Please minimize the use of abbreviations and do not cite references in the abstract

### Data Description
A brief statement providing background and purpose for collection of these data should be presented for readers without specialist knowledge in that area. A clear description of the data, the protocol(s) for data collection, data curation and quality control, as well as potential uses should be included. Please try to keep this concise and close to 1,000 words and 10 references if possible; however, word restrictions may not apply in cases where datasets or data collection/validation protocols are complex or require substantial details to properly describe the work.

Experiments supporting and demonstrating the technical quality of the dataset should be presented. Data Notes can be published alongside traditional Research Articles, and any previous or in-press research using the data should be highlighted, as it will aid validation and demonstrate use. Brief instructions and suggestions of downstream applications that may help other researchers with reuse of the data are also encouraged. Manuscripts that contain extensive biological analyses along with data production should be submitted as a Research Article.

We strongly encourage the inclusion of links to bioinformatics tools that are already publically available. If you are using new bioinformatics tools that you have developed for production or processing of the data described here, we recommend submitting these data and a tool description and test as a Technical Note. Use of workflow management systems such as Galaxy and myExperiment to save the details of the methods to encourage reproducibility as well as conciseness, is also encouraged. Our GigaDB repository, GigaGalaxy server and GitHub page can also be used to archive data, workflows and code with an accompanying DOI, respectively.

### Availability and requirements of software used (if any)
- Link to our [GitHub repository](https://github.com/CAMI-challenge/MetagenomeSimulationPipeline) and the back-to-back Research Article.

### Availability of supporting data and materials
- Archive all simulated reads to [GigaDB](http://gigadb.org/)
